### PR TITLE
update ppd, pdmv, dqm categories

### PR DESCRIPTION
**Pull request by vlimant**
adding @dickychant @rseidita @vlimant 
removing @rvenditti @sunilUIET @kskovpen @jordan-martins @syuvivida @nothingface0 @tjavaid

### CODE REVIEW

**Comment by vlimant** [comment]
FYI @malbouis

File: categories.py [hunk 1]
```diff
@@ -151,11 +148,11 @@
     "forthommel": ["proton-pog"],
     "cardinia": ["tau-pog"],
     "mbluj": ["tau-pog"],
-    "kskovpen": ["pdmv", "tracking-pog"],
+    "kskovpen": ["tracking-pog"],
     "mmasciov": ["tracking-pog"],
     # PPD
     "malbouis": ["ppd"],
-    "jordan-martins": ["ppd"],
+    "vlimant": ["ppd"],
```
Comment (smuzaffar) [inline categories.py:155]: @vlimant , can you please remove this duplicate entry ( it is already there at https://github.com/cms-sw/cms-bot/blob/master/categories.py#L71 )

**Comment by malbouis** [comment]
+1

:-)

**Comment by AdrianoDee** [comment]
+pdmv

**Comment by antoniovagnerini** [comment]
+dqm

**Pull request merged**